tests/qt5_application.py

```diff
@@ -155,72 +155,72 @@ def keyPressEvent(self, e):
             else:
                 self.changing_image_counter += 1
 
-    def closeEvent(self, e):
+    def closeEvent(self, e) -> None:
         self.close()
 
 
 class DragQuitLabel(QtWidgets.QLabel):
 
-    def __init__(self, title, parent):
+    def __init__(self, title: str, parent) -> None:
```

Comment:
Good guess for `title`, the `parent` is `QtWidgets.QWidget`

Reply:
Done. 
there are still undocumented left.

Reply:
That's fine, let's leave this for further Qt5 developments